### ISSUE TITLE
Migrate to New LCHS Go

### ISSUE THREAD

**Issue by awesomehaze** [lchsiteam/lchs-go-pwa#423]
This will involve (in my mind)

- [ ] Change where new go is hosted to go.lciteam.club (this will break the old extension until peoples' browsers pull the new version of the extension)
- [ ] Host this repo at oldgo.lciteam.club
- [ ] Replace the extension with the beta version (done in the developer console)

Celebrate! 🥂

**Comment by mark-crichton** [comment]
fwiw the old repo is currently being hosted both at [go.lciteam.club](https://go.lciteam.club/) and [legacygo.lciteam.club](https://legacygo.lciteam.club/). I'm also currently planning to change new go to go.lciteam.club when ready.
We could either potentially keep a develop branch on [betago.lciteam.club](https://betago.lciteam.club/) or just remove it/redirect back to go.lciteam.club.
As for the extension, I actually do remember pushing a draft update that changed the url to legacygo.lciteam.club, as well as warning in the extension description that the extension was outdated. We could either go with that or replace it entirely with the new go (i personally prefer doing the former but can do the latter if it's more desired).

**Comment by awesomehaze** [comment]
I think it'd be best to push that update which points it to legacygo then wait for people to update. Then we could just make another update once the developer dashboard says enough people are on the new version.

**Comment by mark-crichton** [comment]
Alright, update has been submitted for review. I'm also going to push out a quick update on the new extension so that it no longer shows it is in beta in the extension name.

**Comment by awesomehaze** [comment]
How are we going to move users of the current beta extension to the new one? Ideally it's clickless because lots of users don't know how to install extensions (and have had it added by a friend/family member)

**Comment by mark-crichton** [comment]
looking it up on a search engine, I found a stackoverflow post that details a possible process for it: [https://stackoverflow.com/questions/34307355/how-to-replace-upgrade-a-chrome-extension-with-another](https://stackoverflow.com/questions/34307355/how-to-replace-upgrade-a-chrome-extension-with-another)

Unfortunately, it doesn't seem like extensions can install extensions. However, they can uninstall themselves, and other extensions can uninstall other extensions (if they declare the `management` permission in the manifest).
I was also envisioning having some sort of static banner at the top of the extension page saying that it is outdated and giving a link to the new extension for the user to install themselves (not sure if that'll be too complicated on the user's end tho).

**Comment by Apricitine** [comment]
do u think i should start putting up flyers advertising new go?
i think that would be a great step

**Comment by awesomehaze** [comment]
Good idea! In general it sounds like recruitment is something to work on

Oct 3, 2023 1:33:19 PM Tine ***@***.***>:
